build.sh

```diff
@@ -99,5 +99,5 @@ else
     cmake $CC_PREFIX -DCMAKE_BUILD_TYPE=Release ..
 fi
 
-make |& tee build.log
+make -j $(nproc) |& tee build.log
```

Comment:
Multi-core compile option (make) is really good for building from scratch, building after a small change etc.  IMO, the output is not desirable while updating a common header file with template of a template inside. It could be annoying to update `build.sh` to see what's going on.